src/transaction.ts

```diff
@@ -221,8 +221,8 @@ export class Snapshot extends EventEmitter {
     this._options = {readOnly};
   }
 
-  begin(): BeginPromise;
-  begin(callback: s.BeginTransactionCallback): void;
+  begin(): Promise<BeginResponse>;
```

Comment:
Can we mirror the callback version and call this `BeginTransactionResponse`?

Reply:
You got it 👍